### ISSUE TITLE
The "test-commands" miss checking value's type

### ISSUE THREAD

**Issue by starnight** [flathub/flatpak-builder-lint#94]
When [flathub's CI tries to build](https://buildbot.flathub.org/#/builders/35/builds/1075) `com.hack_computer.Clubhouse`, it shows `test-commands` of the manifest "com.hack_computer.Clubhouse.json" should be a **string**:
```
$ flatpak run --command=flatpak-builder-lint org.flatpak.Builder --exceptions com.hack_computer.Clubhouse.json
{
    "errors": [
        "jsonschema-validation-error"
    ],
    "jsonschema": [
        "['meson test --timeout-multiplier=10 --print-errorlogs'] is not of type 'string'"
    ]
}

If you think problems listed above are a false positive, please report it here:
  https://github.com/flathub/flatpak-builder-lint
```
However, according to `test-commands`' [description](https://github.com/flathub/flatpak-builder-lint/blob/13bce7481526fb40c882975858e3e3ad21f0729e/flatpak_builder_lint/staticfiles/flatpak-manifest.schema.json#L438): "Array of commands to run during the tests.", the value should be **an array of strings**.
